index.bs

```diff
@@ -9,6 +9,7 @@ Mailing List Archives: https://lists.w3.org/Archives/Public/public-script-coord/
 Repository: heycam/webidl
 !Feedback: <a href="https://github.com/heycam/webidl">GitHub</a> (<a href="https://github.com/heycam/webidl/issues/new">new issue</a>, <a href="https://github.com/heycam/webidl/issues">open issues</a>, <a href="https://www.w3.org/Bugs/Public/buglist.cgi?product=WebAppsWG&amp;component=WebIDL&amp;resolution=---">legacy bug tracker</a>)
 ED: https://heycam.github.io/webidl/
+TR: https://www.w3.org/TR/webidl-2/
```

Comment:
The short answer is yes. Looking to see if bikeshed can accommodate https://github.com/tabatkins/bikeshed/issues/1064 .